packages/frameworks/src/frameworks.ts

```diff
@@ -0,0 +1,1605 @@
+          return join(base, content[0]);
+        }
+      } catch (error) {
+        console.error(`Error detecting output directory: `, error);
+      }
+      return base;
+    },
+  },
+  {
+    name: 'Preact',
+    slug: 'preact',
+    demo: 'https://preact.now-examples.now.sh',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/preact.svg',
+    tagline:
+      'Preact is a fast 3kB alternative to React with the same modern API.',
+    description: 'A Preact app, created with the Preact CLI.',
+    website: 'https://preactjs.com',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"preact-cli":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `preact build`',
+      },
+      devCommand: {
+        value: 'preact watch --port $PORT',
+      },
+      outputDirectory: {
+        value: 'build',
+      },
+    },
+    dependency: 'preact-cli',
+    devCommand: 'preact watch --port $PORT',
+    buildCommand: 'preact build',
+    getOutputDirName: async () => 'build',
+    defaultRoutes: [
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        dest: '/index.html',
+      },
+    ],
+  },
+  {
+    name: 'Dojo',
+    slug: 'dojo',
+    demo: 'https://dojo.now-examples.now.sh',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/dojo.png',
+    tagline: 'Dojo is a modern progressive, TypeScript first framework.',
+    description:
+      "A Dojo app, created with the Dojo CLI's cli-create-app command.",
+    website: 'https://dojo.io',
+    detectors: {
+      some: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"@dojo\\/framework":\\s*".+?"[^}]*}',
+        },
+        {
+          path: '.dojorc',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `dojo build`',
+      },
+      devCommand: {
+        value: 'dojo build -m dev -w -s -p $PORT',
+      },
+      outputDirectory: {
+        value: 'output/dist',
+      },
+    },
+    dependency: '@dojo/cli',
+    devCommand: 'dojo build -m dev -w -s -p $PORT',
+    buildCommand: 'dojo build',
+    getOutputDirName: async () => join('output', 'dist'),
+    defaultRoutes: [
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/service-worker.js',
+        headers: { 'cache-control': 's-maxage=0' },
+        continue: true,
+      },
+      {
+        src: '/(.*)',
+        dest: '/index.html',
+      },
+    ],
+  },
+  {
+    name: 'Ember',
```

Comment:
```suggestion
    name: 'Ember.js',
```

Was this intentionally changed?

Reply:
Updated, but to answer your question, these were the values from `static-build`.